qiskit/test/mock/fake_openpulse_2q.py

```diff
@@ -65,7 +65,10 @@ def __init__(self):
             conditional_latency=[
                 [100, 1000], [1000, 100], [100, 1000],
                 [1000, 100], [100, 1000], [1000, 100]
-            ]
+            ],
+            hamiltonian={
+                'h_latex': "MOCK_HAMILTONIAN_STR"
```

Comment:
This is not valid, could you please add a valid Hamiltonian which should include `h_str`, `h_latex` and `vars`, `qub`. See this example for more info https://github.com/Qiskit/qiskit-aer/blob/6c2dd4ed7c42c9d6775293d9832b6779fb3abed9/example/pulse_sim.ipynb.